### ISSUE TITLE
Rule for indentation in fenced code blocks

### ISSUE THREAD

**Issue by polarathene** [DavidAnson/markdownlint#312]
Ideally this could be configured based on language/context given to the code block syntax (\`\`\``language`). Docs that I contribute that require tabs for their config snippets prevents adopting markdownlint otherwise.

Related: https://github.com/DavidAnson/markdownlint/issues/17 https://github.com/DavidAnson/markdownlint/issues/82

**Comment by DavidAnson** [comment]
Can you please clarify the part about “prevents adopting markdownlint”? You can allow tabs in code blocks.

**Comment by polarathene** [comment]
While the markdown documents have tabbed indentation for code snippets, contributions are reviewed to ensure that happens. `markdownlint` lacks a rule to detect the issue so that it can be caught and fixed automatically, so reviewers would still need to check for such themselves right?

**Comment by DavidAnson** [comment]
Yes, in the context of the feature you are requesting, tabs would not be required in code blocks. But that should not prevent you from using markdownlint - it’s more of a missing validation than a wrong or problematic one.

**Comment by polarathene** [comment]
I understand that, but if it cannot help with that issue, the maintainers might not find it acceptable. There's a linked issue where I've pitched `markdownlint`, perhaps they will still express interest despite the lack of such feature.

**Comment by DavidAnson** [comment]
Understood. FYI, this could be implemented as a custom rule by a motivated party - no need to wait for me. :)
